addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -189,6 +189,12 @@ def _hook_invoice_document_after_pdf_report_render(self, invoice, invoice_data):
         reader_buffer.close()
         writer_buffer.close()
 
+    def _get_additional_document_reference_custom_nodes(self, invoice):
+        edi_model = invoice.partner_id._get_edi_builder()
+        if hasattr(edi_model, "_get_additional_document_reference_custom_nodes"):
```

Comment:
I'm not fan of this, I suppose you don't want to pollute the `account.edi.common` model?

Reply:
I thought it might be an easy way of doing it without having to set empty template method.
Do you think I should go for empty template anyway?

Reply:
Do you think that we'll need this function for other edi in the future?

Reply:
I don't know.
Atm, it seems only oioubl needed it.

I thought the ubl 2.0 implementation was fully done but talking with LAS yesterday made me think there are missing stuff in UBL 2.0

For example, it seems the building number is a real thing in ubl 2.0.
The payment means code that I put at the top of the oioubl file seems to be the exact same code for ubl 2.0 and the zatca edi.